roles/nvidiadl_ssd_benchmark_deploy/files/nvidia-pod.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nvidiadl
```

Comment:
could you rename the pod to a name more meaningful, eg `run-ssd-benchmark`?
and a namespace is missing here, it's mandatory for running in the CI (but it works in a local cluster, that's a common source of PR failure ;-)
double check how we do in GPU-burn, either create a dedicated project/ns or use `default`

Reply:
does nvidiadl` means something that I missed?

Reply:
I think it means nvidia deep learning (since this is the NVIDIA deep learning examples SSD code)

Reply:
thanks, I was reading it as "nvidia download" since the very beginning, when it was just a playground Pod, and I got confused to see that all around the PR ;-)
I'm not sure what's the exact link between nvidiadl & mlperf, ie what is best for the name prefix ...

Reply:
The MLPerf and Nvidia Deep Learning models are totally separate, this is just for the Nvidia Deep Learning SSD Training model. Maybe it does read as Nvidia download though... if there's any other naming suggestions let me know! That was just the first thing that came to mind. Writing out "deep learning" seems a bit lengthy